src/main/java/rps/Game.java

```diff
@@ -41,45 +39,45 @@ public void play() {
     }
 
     void playSingleRound() {
-        String status = evaluate(
-                getGestureFrom(PLAYER_ONE_INDEX),
-                getGestureFrom(PLAYER_TWO_INDEX)
+        evaluate(
+                getGestureFromHumanPlayer(),
+                getGestureFromRandomPlayer()
         );
-
-        prompt.display(status);
     }
 
-    String evaluate(Gesture gesture1, Gesture gesture2) {
+    void evaluate(Gesture gesture1, Gesture gesture2) {
```

Comment:
Prefer natural language over numbers, or rather don't mix them up:

`void evaluate(Gesture firstGesture, Gesture secondGesture)`

as opposed to

`void evaluate(Gesture gesture1, Gesture gesture2)`